socrates/lib/site/views/index.jade

```diff
@@ -1,13 +1,13 @@
 extends ../../../views/layout
 block title
-  | SoCraTes 2015 - 5th International Software Craftsmanship and Testing Conference, 27-30 August 2015, Soltau, Germany
+  | SoCraTes 2016 - 5th International Software Craftsmanship and Testing Conference, 25-28 August 2016, Soltau, Germany
 block before-container
   .jumbotron.stage-background
     .container
-      img.logo-left(src='img/socrates2015_logo.png', alt='SoCraTes Conference 2015 - Software Craftsmanship and Testing')
-      h1 SoCraTes 2015
+      img.logo-left(src='img/socrates2015_logo.png', alt='SoCraTes Conference 2016 - Software Craftsmanship and Testing')
```

Comment:
The new logo is available in the Softwerkskammer-Logo-Repo: https://github.com/softwerkskammer/softwerkskammer-logos/tree/master/SoCraTes/2016